WeScan/Common/CaptureSession.swift

```diff
@@ -16,15 +16,15 @@ final class CaptureSession {
     /// Whether the user is past the scanning screen or not (needed to disable auto scan on other screens)
     var isEditing: Bool
     
-    /// Whether auto scan is enabled or not
-    var autoScanEnabled: Bool
+    /// The status of "auto mode". Auto mode does not allow tap to focus and tries to automatically scan a detected rectangle if it has a high enough accuracy.
+    var isAutoModeEnabled: Bool
```

Comment:
Renaming this to `isAutoFocusEnabled` would improve the understanding of the variable

Reply:
Hey, as per the comment auto mode is actually both auto scan and auto focus, so renaming it to `isAutoFocusEnabled` wouldn't make sense IMO.

I did consider separating the 2 modes, but I felt it would be difficult to represent what each icon meant on the limited space available on the toolbar (on the scanner screen).

Reply:
Renamed to isAutoScanEnabled as auto mode no longer matters for auto focus.